README.md

```diff
@@ -80,6 +80,35 @@ mysql -h localhost -D jackal -u jackal -p < mysql.sql
 
 Your database is now ready to connect with jackal.
 
+### Using PostgreSQL
+
+Create a user and a database for that user:
+
+```sql
+CREATE ROLE jackal WITH LOGIN PASSWORD 'password';
+CREATE DATABASE jackal WITH OWNER jackal;
```

Comment:
I wonder if it would be best to have this owned by some other user? I assume jackal will be the user accessing the DB on behalf of the server, so restricting what permissions it has would probably be good (in case of a vulnerability where someone could trick the server into executing a table or DB drop).

Reply:
There are many things that should be done for proper production database setup, but is setting up production environment within the scope of a quick start readme?

Reply:
I'd say "yes"; if it's worth mentioning at all it's worth doing right, otherwise people will just copy/paste, bad things will happen, and ortuman or someone will be blamed later :)

Reply:
Changed it to permissions instead of ownership, but I'm not sure if this actually improves things - if you can SQL-inject you can still do a DROP TABLE or even simply DELETE FROM. Let me know if you have any suggestions on how to improve this further.

Reply:
I don't think you can do a drop table can you? I might be wrong; but you're right, you could always just delete or truncate. it's not likely to happen anyways, but it still seems like good practice to not use the owner as regular user accessing it.

Thanks!

If you really want to add a more robust security model in the database layer later, look up Row level security or row security. It's a bit of a pain to use (and probably isn't worth it), but it's nice to know how to use sometimes.